src/coreclr/vm/method.cpp

```diff
@@ -253,6 +253,32 @@ HRESULT MethodDesc::SetMethodDescVersionState(PTR_MethodDescVersioningState stat
     return S_OK;
 }
 
+#ifdef FEATURE_INTERPRETER
+HRESULT MethodDesc::SetCallStub(CallStubHeader *pHeader)
+{
+    WRAPPER_NO_CONTRACT;
+
+    HRESULT hr;
+    IfFailRet(EnsureCodeDataExists(NULL));
+
+    _ASSERTE(m_codeData != NULL);
+    if (InterlockedCompareExchangeT(&m_codeData->CallStub, pHeader, NULL) != NULL)
+        return S_FALSE;
+
+    return S_OK;
+}
+
+CallStubHeader *MethodDesc::GetCallStub()
+{
+    WRAPPER_NO_CONTRACT;
```

Comment:
```suggestion
    LIMITED_METHOD_CONTRACT;
```

Reply:
Again, I've made it the same as the GetMethodDescVersionState. But why would we want to have the LIMITED_METHOD_CONTRACT when this method just reads the field?

Reply:
`WRAPPER_NO_CONTRACT` is meant to be only used on trivial wrappers when one is too lazy to spell the actual contract.

`LIMITED_METHOD_CONTRACT` is more informative.

Reply:
Ok, makes sense.